service/src/test/java/greencity/ModelUtils.java

```diff
@@ -4467,4 +4468,20 @@ public static UserProfileCreateDto getUserProfileCreateDto() {
             .uuid("f81d4fae-7dec-11d0-a765-00a0c91e6bf6")
             .build();
     }
+
+    public static Order getTestOrderNotTakingBag() {
+        return Order.builder()
+            .id(1L)
+            .orderStatus(OrderStatus.NOT_TAKEN_OUT)
+            .reasonNotTakingBagDescription("Some description")
+            .imageReasonNotTakingBags(List.of("image1", "image2"))
+            .build();
+    }
+
+    public static ReasonNotTakingBagDto getReasonNotTakingBagDtoTest() {
```

Comment:
it is not a test
please remove `test` from the method name

Reply:
done